.github/workflows/ci.yml

```
@@ -23,7 +23,7 @@ jobs:
       -
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 18.18.2
```

```suggestion
          node-version: 18
```

I would suggest keeping this `18` (which is the latest 18 release) so it doesn't need to be managed unless it needs to be updated to a newer major version (e.g. 20)